src/js/providers/html5.ts

```diff
@@ -41,22 +44,32 @@ const HTML5_BASE_WARNING = 324000;
 
 const clearTimeout = window.clearTimeout;
 const _name = 'html5';
-const noop = function () {};
+const noop: () => void = function (): void { /* noop */ };
 
-function _setupListeners(eventsHash, videoTag) {
+function _setupListeners(eventsHash: GenericObject, videoTag: HTMLVideoElement): void {
```

Comment:
We might allow audio tags soon. Guess we could save the renaming and typing for that to be official:  `media: HTMLMediaElement`